src/Pim/Bundle/CatalogBundle/Resources/elasticsearch/index_configuration.yml

```diff
@@ -28,6 +28,12 @@ mappings:
                     mapping:
                         type: 'keyword'
                         normalizer: 'varchar_normalizer'
+            -
+                decimal:
+                    path_match: 'values.*-decimal.*'
```

Comment:
after our discussion, seems good no?
what should we care attention to @SamirBoulil?